portal/server/controllers/utilisation-report-service/utilisation-report-upload/index.js

```diff
@@ -31,6 +31,14 @@ const getUploadErrors = (req, res) => {
       },
     ];
     uploadValidationError = { text: 'You must upload a file' };
+  } else if (res?.locals?.virusScanFailed) {
+    uploadErrorSummary = [
+      {
+        text: 'The selected file could not be uploaded - try again',
+        href: '#utilisation-report-file-upload',
+      },
+    ];
+    uploadValidationError = { text: 'The selected file could not be uploaded – try again' };
```

Comment:
Do you think it's worth making this error message more explicit that it failed the virus scan?

Reply:
If it's essentially a server error in back end I don't really want to give away what's going on/wrong when it might be that nothing is wrong with the file so have left it as generic for now